tests/SalesOrdersClientTest.php

```diff
@@ -44,4 +44,35 @@ public function testCreateAndActivateOrder()
             'expected order ID'
         );
     }
+
+    public function testCreateAndActivateOrderForAddon() {
+        // Setup the client
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new SalesOrdersClient($environment);
+
+        // Create the request
+        $req = new CreateAndActivateOrderRequest();
+        // Create the line items
+        $gSuite = SalesOrdersUtils::buildLineItem('MP-6XDHVMQ84K4THNNP2Z7W2GC28VLHRC4Q');
```

Comment:
There needs to be a lineItem for the App to activate the addon? Or is this example for activating both at the same time

Reply:
Looks like we don't need to include a line item if we are just activating the add-on. I copied the request made from the UI which did that, but testing shows it isn't necessary.